fe1-web/ingestion/handlers/RollCall.ts

```diff
@@ -28,16 +28,14 @@ function handleRollCallCreateMessage(msg: Message): boolean {
 
   const rcMsgData = msg.messageData as CreateRollCall;
 
-  const ongoing = (!!rcMsgData.scheduled);
-
   const rc = new RollCall({
     id: rcMsgData.id,
     name: rcMsgData.name,
     location: rcMsgData.location,
-    description: rcMsgData.roll_call_description,
+    description: rcMsgData.description,
     creation: rcMsgData.creation,
-    start: ongoing ? rcMsgData.start : rcMsgData.scheduled,
-    ongoing: ongoing,
+    start: rcMsgData.proposed_start,
```

Comment:
As discussed with @pierluca on Slack, we set the start field of the RollCall object to proposed_start

Reply:
I'm not going to bother you with this within this PR , but at some point you might need to change (for display reasons) the `ongoing` field to a state field, containing an enum giving you the state (CREATED/OPENED/CLOSED/REOPENED), as otherwise you don't know if start/end represent proposed values or actual values :)

Reply:
(another approach could have been to keep the `proposed_start` , `proposed_end` , and alias them to `start` ... but you'd still need a state field :) )

Reply:
Good idea. I have now done the state field and am think that the clearest solution would be to have all 4 fields in the object:
- proposed_start
- proposed_end
- started_at
- closed_at
And therefore not use the `RollCall implements LaoEvent {`. 
Would that be ok or is it a must that it implements the LaoEvent?

Reply:
You can address that by using getters to implement `LaoEvent`
`LaoEvent` and `LaoEventState` (if I remember correctly the name) are used to handle events in a generic way, both in the user interfaces and in the reducer (eventReducer , if I also remember correctly).

Reply:
This is a very elegant solution, thanks.